pages/trending.tsx

```diff
@@ -37,9 +42,11 @@ const Trending = ({ data, query }) => {
         description="See the libraries that are trending today.">
         <View style={{ width: 160, marginHorizontal: 'auto', marginTop: 12 }}>
```

Comment:
```suggestion
        <View style={{ marginHorizontal: 'auto', marginTop: 12 }}>
```

After adding `width` to the Button itself, the `width` from container need to be removed, since Filter button in active state overflows the container, and becomes places slightly to the right from the center.